locale/ru_RU.ts

```diff
@@ -1,388 +1,388 @@
+<TS language="ru" version="2.1">
 <context>
     <name>About</name>
     <message>
-        <location filename="../about.ui" line="17"/>
+        <location line="17" filename="../about.ui"/>
         <source>About</source>
         <translation>О программе</translation>
     </message>
     <message>
-        <location filename="../about.ui" line="66"/>
+        <location line="66" filename="../about.ui"/>
         <source>GoldenDict dictionary lookup program, version </source>
         <translation>Словарь GoldenDict, версия </translation>
     </message>
     <message>
-        <location filename="../about.ui" line="91"/>
+        <location line="91" filename="../about.ui"/>
         <source>(c) 2008-2013 Konstantin Isakov (ikm@goldendict.org)</source>
         <translation>© Константин Исаков (ikm@goldendict.org), 2008-2013</translation>
     </message>
     <message>
-        <location filename="../about.ui" line="125"/>
+        <location line="125" filename="../about.ui"/>
         <source>Credits:</source>
         <translation>Список участников:</translation>
     </message>
     <message>
-        <location filename="../about.ui" line="101"/>
+        <location line="101" filename="../about.ui"/>
         <source>Licensed under GNU GPLv3 or later</source>
         <translation>Лицензия: GNU GPLv3 или более поздняя версия</translation>
     </message>
     <message>
-        <location filename="../about.cc" line="19"/>
+        <location line="19" filename="../about.cc"/>
         <source>[Unknown]</source>
         <translation>[Неизвестно]</translation>
     </message>
     <message>
-        <location filename="../about.cc" line="36"/>
+        <location line="36" filename="../about.cc"/>
         <source>Based on Qt %1 (%2, %3 bit)</source>
         <translation>На базе Qt %1 (%2, %3 бит)</translation>
     </message>
 </context>
 <context>
     <name>ArticleMaker</name>
     <message>
-        <location filename="../article_maker.cc" line="210"/>
+        <location line="210" filename="../article_maker.cc"/>
         <source>Expand article</source>
         <translation>Развернуть статью</translation>
     </message>
     <message>
-        <location filename="../article_maker.cc" line="216"/>
+        <location line="216" filename="../article_maker.cc"/>
         <source>Collapse article</source>
         <translation>Свернуть статью</translation>
     </message>
     <message>
-        <location filename="../article_maker.cc" line="245"/>
-        <source>No translation for &lt;b&gt;%1&lt;/b&gt; was found in group &lt;b&gt;%2&lt;/b&gt;.</source>
-        <translation>В группе &lt;b&gt;%2&lt;/b&gt; не найдено перевода для &lt;b&gt;%1&lt;/b&gt;.</translation>
+        <location line="245" filename="../article_maker.cc"/>
+        <source>No translation for &lt;b>%1&lt;/b> was found in group &lt;b>%2&lt;/b>.</source>
+        <translation>В группе &lt;b>%2&lt;/b> не найдено перевода для &lt;b>%1&lt;/b>.</translation>
```

Comment:
Replace `>` back to `&gt;`.

Reply:
Added the fixed file again. It seems opening it in Lokalize changes &gt; to >.